### ISSUE TITLE
Update visual design of header

### ISSUE THREAD

**Issue by astridu** [sul-dlss/argo#4529]
Current header:
<img width="1643" alt="image" src="https://github.com/sul-dlss/argo/assets/42216982/7feadf2f-49b5-4a3c-beb3-9a2d6c67602b">

New header (for reference, this is the same style as the [Preassembly header](https://sul-preassembly-prod.stanford.edu/) that was just updated):

<img width="718" alt="image" src="https://github.com/sul-dlss/argo/assets/42216982/90b29756-fa25-4cdc-a0f3-a65795d5431e">

- Background color: Stone/Dark 544948
- Stanford University in cardinal red 8C1515 at the top on white background: links to www.stanford.edu
- Stanford Libraries logo (white text w/full color rosette): links to library.stanford.edu
- Top right nav: Feedback and Logout (goes to the same place they do today); white links with underline on hover
- Title: Argo (Semibold 36pt)
- Bottom right nav: Impersonate   •.  All workflows   •.  Bulk action   •   Register (Bold with white link on hover, there should be about 3 spaces between each link and the bullet point separating the 4 links)

**Comment by astridu** [comment]
Also an example of our header template here if this helps: https://sul-dlss.github.io/component-library/header/

**Comment by jcoyne** [comment]
@astridu where would you put the "Logged in as <name>" in the new design? Are we going to keep the register drop down, or does that functionality move elsewhere? 
<img width="199" alt="Screenshot 2024-07-10 at 8 51 53 AM" src="https://github.com/sul-dlss/argo/assets/92044/9e1e4822-d8d0-4d21-82e0-25d5875b27db">

**Comment by astridu** [comment]
1. I didn't think we needed the 'logged in as', but let me know if there is a compelling reason to keep it. 
2. Sorry, the mockup was just missing the drop down icon.
<img width="716" alt="image" src="https://github.com/sul-dlss/argo/assets/42216982/8ea00f31-fb96-458b-8fc7-4a00eb4c6d87">

**Comment by justinlittman** [comment]
Is this design supposed to be responsive? (And if so, how?)

**Comment by astridu** [comment]
Yes. This is what we have, but I need to doublecheck on the mobile view bc we are currently working on that one. 
<img width="800" alt="image" src="https://github.com/sul-dlss/argo/assets/42216982/1fcc97be-253b-4b3f-a0a3-20c07251497c">

**Comment by jcoyne** [comment]
@astridu the "logged in as" is important if you're impersonating some other user/group.

**Comment by astridu** [comment]
ok, will work on it.

**Comment by astridu** [comment]
I added the 'logged in as' back. It's next to the logout link. As for making this responsive, the EarthWorks team is working on this same header and will have the responsive design. I'm wondering if you want to wait until they have it completed and copy it? 
<img width="717" alt="image" src="https://github.com/user-attachments/assets/c4c704e7-9f7d-447f-bcdc-8e2a28076048">

**Comment by jcoyne** [comment]
We should be able to just replace the `palo-alto-dark` class we have in the component library with a `stone-dark` class described as:
```
:root {
  --stanford-stone-dark-rgb: 84, 73, 72;
}
.stone-dark {
  --bs-dark-rgb: var(--stanford-stone-dark-rgb);
}
```

**Comment by aaron-collier** [comment]
@astridu What font is used for "Argo" in the header? When using the component library and styles, I get something slightly different: 

<img width="275" alt="Screenshot 2024-07-16 at 2 53 31 PM" src="https://github.com/user-attachments/assets/1b3b1264-a0ce-47b4-a3a3-d10c4411d6ff">

Thanks.

**Comment by astridu** [comment]
@aaron-collier Good catch! That is definitely not the right font. It should be "Source Sans 3", semibold, size 36. Thank you for letting me know.

**Comment by jcoyne** [comment]
@astridu that looks like Source Sans 3 to me.  How can you tell?

**Comment by aaron-collier** [comment]
@jcoyne what I noticed was that the lower case g is different.
